src/MediaWiki/Renderer/HtmlFormRenderer.php

```
@@ -232,55 +174,39 @@ public function addNonBreakingSpace(): static {
 	}
 
 	/**
 	 * @since 3.0
-	 *
-	 * @param string $element
-	 * @param array $attributes
-	 *
-	 * @return HtmlFormRenderer
 	 */
-	public function closeElement( $element = 'div', array $attributes = [] ): static {
+	public function closeElement( string $element = 'div', array $attributes = [] ): static {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)